lib/user-interface/react/src/components/chatbot/Chat.tsx

```diff
@@ -293,6 +281,24 @@ export default function Chat({ sessionId }) {
     };
   };
 
+  const createOpenAiClient = (streaming: boolean) => {
+    return new ChatOpenAI({
+      modelName: selectedModel?.id,
+      openAIApiKey: auth.user?.id_token,
+      configuration: {
+        baseURL: `${RESTAPI_URI}/v2/serve`,
```

Comment:
This should probably be using the `RESTAPI_VERSION` variable?

Reply:
I had these set to `RESTAPI_VERSION` but @petermuller had some concern about customers having to know to update their configuration files when doing the redeploy and that potentially causing issues.

Reply:
Talked with peter, I think we are good with keeping `RESTAPI_VERSION` - adding it back to the URLs